Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.7-alpine
+
+WORKDIR /app
+ADD . /app
+
+RUN apk update && apk add linux-headers postgresql-dev gcc python3-dev musl-dev
+
+RUN python3 -m venv env
+RUN source env/bin/activate
+RUN pip install -r requirements.txt
+

```

Comment:
I had to add `CMD python manage.py db migrate && python manage.py db upgrade` to migrate the database.

Reply:
These should have been running on the `app-entrypoint.sh` so I didn't add these in, is yours not running on the entrypoint?